pyvisa-py/tcpip.py

```diff
@@ -91,7 +91,7 @@ def read(self, count):
         else:
             term_char = 0
 
-        if self.term_char:
+        if hasattr(self, 'term_char') and self.term_char:
```

Comment:
This should actually be 
```python
if term_char
```
The variable defined in 90 or 92

The same in line 96